app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,6 @@
     <color name="colorRed">#F5060B</color>
     <color name="colorEcruWhite">#F6F7ED</color>
     <color name="colorMerino">#F9F5F0</color>
+    <color name="colorBackground">#F8F5F1</color>
```

Comment:
http://chir.ag/projects/name-that-color/#F8F5F1
```suggestion
    <color name="colorSpringWood">#F8F5F1</color>
```

Reply:
http://chir.ag/projects/name-that-color
Dùng cái này để lấy tên mã màu nha